libp2p/NodeTable.cpp

```diff
@@ -299,7 +295,8 @@ void NodeTable::evict(shared_ptr<NodeEntry> _leastSeen, shared_ptr<NodeEntry> _n
 
     if (evicts == 1)
         doCheckEvictions();
-    ping(_leastSeen.get());
+    if (_leastSeen)
```

Comment:
I think this `if` is not needed, it's already checked at the calling side, and `_lastSeen` is already used above in this function anyway

Reply:
@gumb0 Can you fix it now? I'd like to make a build with aleth-bootnode included.